src/parser/core/modules/Defensives.tsx

```diff
@@ -124,15 +130,16 @@ export class Defensives extends Analyser {
 							content: {
 								content: <Table compact unstackable celled>
 									<Table.Body>
-										{
-											this.tryGetAdditionalUseRow(defensive)
-										}
+										{firstAdditionalUsageTry}
```

Comment:
It seems like the changes we made recently to better clamp around the actual pull start and end times might be related to the prepull issues we're seeing here. There's some code in getAdditionalUsageData that is trying to adjust the availableTimestamp based on whether there was a prepull use of the action or not, and that's now causing the problems we're seeing as a side effect.

Instead of this change to check for prepull and skip the tryGetAdditionalUseRow, I think we should remove that availableTimestamp adjustment in getAdditionalUsageData. In my testing, that fixes both the prepull row duplication issue, and the other issue reported around mid-fight available windows not showing properly.

Reply:
oooh okie! would you like me to include that in this PR or do a separate one since it's more of a fix for both?

Reply:
after looking at removing the available timestamp adjustment, it doesn't quite fix the duplicate issue
let me do a separate PR for this specific issue so we can address both

(e.g. if I comment out that adjustment, I see the fix for the missing suggestion, but I don't see a fix for the duplicate noted by that other commentor)
https://discord.com/channels/441414116914233364/470049935077933076/1275859622196150423
Here is what it looks like in dev after commenting out the available timestamp adjustment
![image](https://github.com/user-attachments/assets/ecf7dc5e-5711-494d-9471-cf55f1f132fe)

Reply:
nvm I don't know what I'm doing lol I see something about clamps and got lost lol

Reply:
fyi, this will be removed from this PR as it's addressed independently in #2100